pkg/encoder/raw_syscall_enter_windows.go

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package encoder
+
+import "github.com/cilium/tetragon/api/v1/tetragon"
+
+func rawSyscallEnter(tp *tetragon.ProcessTracepoint) string {
+	return "unknown win32"
+
```

Comment:
why win32? unknown or 'not supported' would do?

Reply:
That is generally what syscalls on Windows are called: win32 APIs